tests/tests_base.py

```diff
@@ -0,0 +1,124 @@
+import codecs
+from PIL import Image
+import sys
+sys.path = ["src"] + sys.path
+
+from pyocr import builders
+from pyocr import tesseract
+
+
+class BaseTest(object):
+    tool = None
+
+    def _path_to_img(self, image_file):
+        raise NotImplementedError("Implement in subclasses.")
+
+    def _path_to_out(self, expected_output_file):
+        raise NotImplementedError("Implement in subclasses.")
+
+    def _read_from_expected(self, expected_output_path):
+        raise NotImplementedError("Implement in subclasses.")
+
+    def _read_from_img(self, image_path, lang=None):
+        return self.tool.image_to_string(
+            Image.open(image_path),
+            lang=lang,
+            builder=self._builder
+        )
+
+    def _test_equal(self, output, expected_output):
+        raise NotImplementedError("Implement in subclasses.")
+
+    def set_builder(self):
+        raise NotImplementedError("Implemented in subclasses.")
+
+    def setUp(self):
+        self.set_builder()
+
+    def _test_txt(self, image_file, expected_output_file, lang='eng'):
+        image_path = self._path_to_img(image_file)
+        expected_output_path = self._path_to_out(expected_output_file)
+        
+        expected_output = self._read_from_expected(expected_output_path)
+        output = self._read_from_img(image_path, lang)
+
+        self._test_equal(output, expected_output)
+
+
+class BaseTestText(BaseTest):
+    def set_builder(self):
+        self._builder = builders.TextBuilder()
+
+    def _read_from_expected(self, expected_output_path):
+        expected_output = ""
+        with codecs.open(expected_output_path, 'r', encoding='utf-8') \
+                as file_descriptor:
+            for line in file_descriptor:
+                expected_output += line
+        return expected_output.strip()
+
+    def _test_equal(self, output, expected_output):
```

Comment:
None, it's just to be consistent with the other tests that use more complex assertions. I just noticed that much of the redundancy in the tests came from having different combinations of creating the `output` / `expected_output` and testing the equality between them, so I put in methods for both things.